### ISSUE TITLE
Type '{}' not assignable

### ISSUE THREAD

**Issue by autorunman22** [modbender/nuxt-snackbar#3]
I'm getting this warning on my vue component even though the module works fine:
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/4214419/226779710-a3ec08af-06a1-4cb4-ae9d-9e41a93c1943.png">

It's quite bothering because it makes the file color red.

**Comment by modbender** [comment]
That won't cause a issue but yes I'm aware of it. Next update within the week will fix it.

**Comment by autorunman22** [comment]
For easy fix, just convert your template to PUG:
```
<template lang="pug">
NuxtLayout
  NuxtPage
  NuxtSnackbar
</template>

```

**Comment by modbender** [comment]
Good workaround.

**Comment by modbender** [comment]
@autorunman22 Fixed in v1.0.3

**Comment by autorunman22** [comment]
Thank you @modbender !
